dev/e2e/lib/main.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
maybe move this up one directory and call it keyboard_resize_test.dart or something?